tools/const_finder/bin/main.dart

```diff
@@ -43,10 +43,6 @@ void main(List<String> args) {
         valueHelp: 'path/to/main.dill',
         help: 'The path to a kernel file to parse, which was created from the '
             'main-package-uri library.')
-    ..addOption('main-library-uri',
-        help: 'The package: URI to treat as the main entrypoint library '
-            '(the same package used to create the kernel-file).',
-        valueHelp: 'package:hello_world/main.dart')
```

Comment:
This just doesn't work. Gallery, for example, uses multiple libraries to define icons.

Reply:
Did you add any tests to cover this?

Reply:
You're making me sad.  I'll update the test.

Reply:
Done